pkgs/top-level/stage.nix

```
@@ -269,6 +269,12 @@ let
     });
   };
 
+  # Auto-calls packages declared in the named package hierarchy in pkgs/unit
+  unitPackages = self: super:
+    lib.mapAttrs (name: file:
+      self.callPackage file { }
+    ) (import ./unit-files.nix);
```

```suggestion
  unitPackages = self: super:
    lib.mapAttrs
      (name: file: self.callPackage file { })
      (import ./unit-files.nix);
```

Motivation: arguments to the same curried function have the same indentation.

---

Interesting how you're suggesting this here, that's the exact style me and @piegamesde are going with for NixOS/rfcs#101! Will apply :smile: